examples/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.googleAuth;
+
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.grpc.CallCredentials;
+import io.grpc.auth.MoreCallCredentials;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import com.google.pubsub.v1.PublisherGrpc;
+import com.google.pubsub.v1.ListTopicsResponse;
+import com.google.pubsub.v1.ListTopicsRequest;
+
+import java.io.FileInputStream;
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Example to illustrate use of Google credentials as described in
+ * @see <a href="../../../../../../GOOGLE_AUTH_EXAMPLE.md">Google Auth Example README</a>
+ *
+ * Also @see <a href="https://cloud.google.com/pubsub/docs/reference/rpc/">Google Cloud Pubsub via gRPC</a>
+ *
+ */
+public class GoogleAuthClient {
+  private static final Logger logger = Logger.getLogger(GoogleAuthClient.class.getName());
+
+  private final ManagedChannel channel;
+
+  /**
+   * stub generated from the proto file.
+   */
+  private final PublisherGrpc.PublisherBlockingStub blockingStub;
+
+  /**
+   * Construct our gRPC client that connects to the pubsub server at {@code host:port}.
+   *
+   * @param host  host to connect to - typically "pubsub.googleapis.com"
+   * @param port  port to connect to - typically 443 - the TLS port
+   * @param callCredentials  the Google call credentials created from a JSON file
+   */
+  public GoogleAuthClient(String host, int port, CallCredentials callCredentials) {
+    // Google API invocation requires a secure channel. Channels are secure by default (SSL/TLS)
+    this(ManagedChannelBuilder.forAddress(host, port)
```

Comment:
I believe this can fit on a single line.  Also, we typically break lines at the highest semantic level, which would imply keeping the entire argument to `this()` on a single line.  The `this(` and `)` can be on separate lines though, if needed.